cognee/tasks/repo_processor/get_local_dependencies.py

```
@@ -1,154 +1,116 @@
+    code_file_node = CodeFile(
+        id=uuid5(NAMESPACE_OID, script_path),
+        source_code=None,
+        file_path=relative_file_path,
+    )
+
+    source_code_tree = source_code_parser.parse(bytes(source_code, "utf-8"))
+
+    async for part in extract_code_parts(source_code_tree.root_node):
+        part.file_path = relative_file_path
+
+        if type(part) == FunctionDefinition:
```

Ruff (0.8.2)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)